### ISSUE TITLE
Importing native MLO ytyp throws an error

### ISSUE THREAD

**Issue by MoravianLion** [Skylumz/Sollumz#338]
I was loading in some original ytyp files for MLOs and when trying to work with "v_int_62.ytyp", I've got following error in Blender

```
Error during import: Traceback (most recent call last):
  File "C:\Users\MoravianLion\AppData\Roaming\Blender Foundation\Blender\3.2\scripts\addons\Sollumz-main\ytyp\operators.py", line 634, in run
    room.blend = room_xml.blend
TypeError: bpy_struct: item.attr = val: RoomProperties.blend expected an int type, not float
```

Resulting in missing rooms (besides probably bunch of other things) inside Sollumz's ytyp configuration menu.

I never worked with Python before, so this is the only information I'm currently able to produce.

**Comment by CFox04** [comment]
Update Sollumz. I do not get this error on the latest version and line 634 does not contain the code ``room.blend = room_xml.blend`` anymore, so you're Sollumz is definitely outdated.

**Comment by CFox04** [comment]
Turns out this bug did still exist in the latest version, just at a different place. Not sure why I wasn't getting the error though. It's fixed in commit 6a3f828b3be11b912bf90d3fa59bc476da6210d0
